### PR TITLE
Added vee-validate example

### PR DESCRIPTION
**Pull request by toadkicker**
Added example for vee-validate and fixed example for vuelidate state property

### CODE REVIEW

**Comment by toadkicker** [comment]
Of note I'm excluding the steps on the setup of vee-validate because I felt it was better to leave that with them. I hope that is clear for the example as the vuelidate one skips those steps as well.

File: docs/markdown/reference/validation/README.md [hunk 1]
```diff
@@ -86,3 +86,71 @@ This is a verbose example designed to show how Bootstrap-Vue and Vuelidate inter
...
+        This is a required field and must be at least 3 characters
+      </b-form-invalid-feedback>
+    </b-form-group>
+    <b-form-group id="exampleInputGroup2"
+                  label="Food"
+                  label-for="exampleInput2">
+      <b-form-select id="exampleInput2"
+                     :options="foods"
+                     v-validate="{required: true}"
+                     :state="validateState('form.foods')"
```
Comment (mosinve) [inline docs/markdown/reference/validation/README.md]: Where this method from?

File: docs/markdown/reference/validation/README.md [hunk 2]
```diff
@@ -86,3 +86,71 @@ This is a verbose example designed to show how Bootstrap-Vue and Vuelidate inter
...
+                     v-model="form.food" />
+      <b-form-invalid-feedback id="input2LiveFeedback">
+        This is a required field
+      </b-form-invalid-feedback>
+    </b-form-group>
+    <b-button type="submit"
+              variant="primary"
+              :disabled="$v.form.$invalid">
```
Comment (mosinve) [inline docs/markdown/reference/validation/README.md]: `$v` is `Vuelidate` related, so here must be `vee-validate` form state check

File: docs/markdown/reference/validation/README.md (hunk 1 above)
Comment (toadkicker) [reply docs/markdown/reference/validation/README.md]: Thanks for reviewing. I wrote it, but it didn't end up in the patch. I've updated the doc to include it.

**Pull request merged**